### DISCUSSION TITLE
Entity Texture not loading in-game

### DISCUSSION THREAD

**Discussion by deanwean** [FabricMC/fabric-api#2020] [Mod Dev Support] [answered]
I built my mod, and I tested it out on my Minecraft launcher and it didn't load the texture of my entity.
When I test my mod out in the project its launcher, you can see the texture.
The logs and the debug console don't say anything is wrong.
I have the Fabric API installed.

![image](https://user-images.githubusercontent.com/41543087/155105734-3bab65e7-92a3-46fd-82bb-c6a7157595e3.png)

What could the problem be and how could I possibly find it?
If you need any more information, you can ask because I don't really know what information to post here other than this.

**Comment by warjort** [comment, marked as answer]
The game is showing the "missingno" (purple and black) texture you get when it can't load a model or texture.
I would guess if you look closely in your log/latest.log there is a warning about it?

The difference between development and "production" is that development effectively runs the game with your src/main/resources in the classpath while production uses a jar file which you can find in the build/libs folder of your project.
So check your jar file actually contains the files you expect it to have.
Run the command ````jar -tf your-mod.jar```` or use a tool that can handle zip files to look inside it.
